howso/conviction.amlg

```diff
@@ -112,6 +109,27 @@
 			model_size (call !GetNumTrainingCases)
 		))
 
+		(if (= (null) use_case_weights)
```

Comment:
this exact block of code is used all over, can you please make it into a function where it has the parameter `set_valid_weight_feature`,  when 'set_valid_weight_feature' is true, it'll also include that 4 line `(if use_case_weights` portion as well.  that way you can just call this one method in all the places, even those that don't set valid_weight_feature

Reply:
done!